docs/whatsnew/7.0.rst

```diff
@@ -420,6 +421,32 @@ simple example might be:
     >>> stats.biweight_location(), stats.biweight_scale()  # doctest: +FLOAT_CMP
     (np.float64(67.98055399699436), np.float64(64.82889460022386))
 
+.. _whatsnew-7.0-masked-coordinates:
+
+Support for masks in ``coordinates``
+====================================
+
+Initial support for masks has been implemented for |SkyCoord|, as well as for
+coordinate frames and coordinate representations.
+
+Like more generally for |Masked| instances, one can access the mask using
+`~astropy.coordinates.SkyCoord.mask` and check whether the underlying data are
+masked using `~astropy.coordinates.SkyCoord.masked`. Like for all masked
+classes, one can access underlying unmasked data using
+`~astropy.utils.masked.MaskableShapedLikeNDArray.unmasked` and one can get a
```

Comment:
Sadly, because `SkyCoord` does not show inherited components, I can still not link directly to it for `unmasked` and `filled`.